### DISCUSSION TITLE
Need Help.

### DISCUSSION THREAD

**Discussion by JetXplorer** [MustardChef/WSABuilds#139] [Q&A] [answered]
Hi. I mistakenly deleted the installation folder. What should I do now to reinstall it. I'm a windows 10 user.

Thanks
JetXplorer

**Comment by MustardChef** [comment, marked as answer]
@JetXplorer, Hello.
If you are lucky, you may be able to salvage your apps and data.
Go to ```%LOCALAPPDATA%\Packages\MicrosoftCorporationII.WindowsSubsystemForAndroid_8wekyb3d8bbwe\LocalCache\``` and check if the Userdata.vhdx is intact.

If it there, follow the Restore VHDX Backup section of the Backup/Restore guide in Readme.md.
![image](https://github.com/MustardChef/WSABuilds/assets/68516357/88973bd9-ab6d-4cd8-bfdd-d4e20f2eebba)

**Reply by JetXplorer** [reply]
Yes, it is there.

**Reply by MustardChef** [reply]
Great to hear. Make sure you download the same type of builds as before. For example, if youn downloaded a Magisk build, download a magisk build. It is not recommeded to mix and match builds.

**Reply by JetXplorer** [reply]
Okay, Thank you.

**Reply by JetXplorer** [reply]
What should I do after deleting the lines?

**Reply by MustardChef** [reply]
You should run the Run.bat file.

**Reply by JetXplorer** [reply]
Thank you So MUCH!! It worked!! :) Such a gr8 project by you. Hands off to you 🙋‍♂️
